Shared/Samples/Find route around barriers/FindRouteAroundBarriersView.swift

```diff
@@ -109,22 +90,38 @@ struct FindRouteAroundBarriersView: View {
                             await mapViewProxy.setViewpointGeometry(directionGeometry, padding: 100)
                         }
                         Spacer()
-                        
-                        Picker("Features", selection: $featuresSelection) {
-                            Text("Stops").tag(RouteFeatures.stops)
-                            Text("Barriers").tag(RouteFeatures.barriers)
+                        Button("Get Route") {
+                            routingIsInProgress = true
                         }
-                        .pickerStyle(.segmented)
-                        .labelsHidden()
-                        Spacer()
-                        
-                        SheetButton(title: "Route Settings") {
-                            RouteParametersSettings(for: model.routeParameters)
-                        } label: {
-                            Image(systemName: "gear")
+                        .disabled(model.stopsCount < 2)
+                        .task(id: routingIsInProgress) {
+                            guard routingIsInProgress else { return }
+                            
+                            do {
+                                // Route when the button is pressed.
+                                try await model.route()
+                                
+                                // Update the viewpoint to the geometry of the new route.
+                                guard let geometry = model.route?.geometry else { return }
+                                await mapViewProxy.setViewpointGeometry(geometry, padding: 50)
+                            } catch {
+                                self.error = error
+                            }
+                            
+                            routingIsInProgress = false
                         }
                         Spacer()
+                        Button("Settings", systemImage: "gear") {
+                            showSettings.toggle()
+                        }
+                        .labelsHidden()
+                        .popover(isPresented: $showSettings) {
+                            settings
+                                .presentationDetents([.fraction(0.35)])
+                                .frame(idealWidth: 400, idealHeight: 400)
```

Comment:
@yo1995 do you have any objections to idealWidth being 400, I think the look is better at 400